evalml/automl/automl_search.py

```diff
@@ -409,20 +409,18 @@ def search(self, X, y, data_checks="auto", show_iteration_plot=True):
             logger.warning("`y` passed was not a DataColumn. EvalML will try to convert the input as a Woodwork DataTable and types will be inferred. To control this behavior, please pass in a Woodwork DataTable instead.")
             y = _convert_to_woodwork_structure(y)
 
-        X = _convert_woodwork_types_wrapper(X.to_dataframe())
-        y = _convert_woodwork_types_wrapper(y.to_series())
-
+        # X = _convert_woodwork_types_wrapper(X.to_dataframe())
+        # y = _convert_woodwork_types_wrapper(y.to_series())
         self._set_data_split(X)
 
         data_checks = self._validate_data_checks(data_checks)
-        self._data_check_results = data_checks.validate(X, y)
+        self._data_check_results = data_checks.validate(_convert_woodwork_types_wrapper(X.to_dataframe()), _convert_woodwork_types_wrapper(y.to_series()))
```

Comment:
Data checks currently don't support Woodwork types (to do in #1292 or later PR), so passing pandas to data checks.